lastuser_oauth/views/notify.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 
 import requests
-from flask_rq import job
+from lastuserapp import rq
```

Comment:
We can't import from `lastuserapp` here, since this is a blueprint that is a dependency for the app. Move the RQ object here instead.